variables.tf

```diff
@@ -4,14 +4,35 @@ variable "sysdig_secure_api_token" {
   description = "Sysdig Secure API token"
 }
 
-variable "org_cloudvision_member_account_id" {
-  type        = string
-  description = "the **account id within the organization** to be used as cloudvision account"
-}
 
-variable "org_cloudvision_account_region" {
-  type        = string
-  description = "default cloudvision member account region for services provisioning"
+variable "cloudvision_organizational_setup" {
+  type = object({
+    is_organizational                 = bool
+    connector_ecs_task_role_name      = string
+    org_cloudvision_member_account_id = string
+    org_cloudvision_role              = string
+  })
+  default = {
```

Comment:
for readability, moved `is_organizational` out `of organizational_setup` object block.
this leads to not being able to use valiation blocks as they only make sense if we can check `is_organizational=true`, but cannot query variables outside `this`.
another option would be to use optionals in objects, but not supported https://github.com/hashicorp/terraform/issues/19898